scripts/DGSPlanner/InstrumentSetupWidget.py

```diff
@@ -95,25 +95,22 @@ def setData(self, index, value, role = QtCore.Qt.EditRole):
             row = index.row()
             column = index.column()
             if column<=1:
-                try:
-                    val=str(value.toString()) #QVariant
-                except AttributeError:
-                    val=str(value) #string
+                val=str(value)
                 if column==0:
                     self.labels[row]=val
                 else:
                     self.dirstrings[row]=val
             elif column==2:
                 try:
-                    val=value.toInt()[0] #QVariant
-                except AttributeError:
-                    val=int(value) #string
+                    val=val=int(value)
```

Comment:
Should this line simply be:
```suggestion
                    val=int(value)
```